### ISSUE TITLE
Apply multiple themes

### ISSUE THREAD

**Issue by TobseTest** [spacecowboy/NoNonsense-FilePicker#152]
First off all, thanks for this wonderful library.
I am new to Android development and I would like to know if it is possible to apply multiple themes to the picker.
Because I use the file picker in my settings activity, my class already extends another class.
Thanks for all replies in advance.

**Comment by spacecowboy** [comment]
Yes you can indeed use multiple themes. Themes are applied from your manifest and the way you get multiple themes in different contexts is by defining multiple file pickers in your manifest. 

A clear example of this is in the sample app which demonstrates a dark and a light theme by having one of each of all kinds of pickers.

Notice here in the manifest that two FilePickers are defined with different themes: 

* [dark](https://github.com/spacecowboy/NoNonsense-FilePicker/blob/master/sample/src/main/AndroidManifest.xml#L53)

* [light](https://github.com/spacecowboy/NoNonsense-FilePicker/blob/master/sample/src/main/AndroidManifest.xml#L63)

The light themed picker references a wrapper class just to have a unique name, [no implementation is necessary](https://github.com/spacecowboy/NoNonsense-FilePicker/blob/master/sample/src/main/java/com/nononsenseapps/filepicker/sample/FilePickerActivity2.java).

Themes are defined in [styles.xml](https://github.com/spacecowboy/NoNonsense-FilePicker/blob/master/sample/src/main/res/values/styles.xml).
